### PR TITLE
chaoskube: add chaoskube helm chart

### PR DESCRIPTION
**Pull request by linki**
this adds a helm chart for [chaoskube](https://github.com/linki/chaoskube).

chaoskube is a little tool that periodically kills random pods in your cluster to test how well your applications can deal with failing pods.

### CODE REVIEW

**Comment by viglesiasce** [comment]
@k8s-bot ok to test

File: incubator/chaoskube/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+## Configuration
+
+| Parameter                 | Description                                         | Default                           |
+|---------------------------|-----------------------------------------------------|-----------------------------------|
+| `name`                    | container name                                      | chaoskube                         |
+| `image`                   | docker image                                        | quay.io/linki/chaoskube           |
+| `imageTag`                | docker image tag                                    | v0.3.1                            |
+| `replicas`                | number of replicas to run                           | 1                                 |
+| `interval`                | interval between pod terminations                   | 10m                               |
+| `dryRun`                  | defines whether pods should actually be terminated  | false                             |
```
Comment (viglesiasce) [inline incubator/chaoskube/README.md]: Lets make dryRun on by default to protect users from accidentally shooting themselves (and others in the cluster) in the foot. Obviously that is the whole point of installing the chart but better to be safe than sorry.

**Review by viglesiasce — CHANGES_REQUESTED** [review]
One default change (dryRun->true).

File: incubator/chaoskube/README.md (hunk 1 above)
Comment (linki) [reply incubator/chaoskube/README.md]: I agree and changed it.

**Comment by linki** [comment]
@viglesiasce I changed the default for dryRun to `true` so nobody can accidentally start killing stuff in the cluster.

**Review by viglesiasce — APPROVED** [review]

**Comment by viglesiasce** [comment]
Looks good to me @linki! Do you want to move this to stable since it meets all the requirements?

**Comment by linki** [comment]
@viglesiasce yes, I moved it. Thanks for your review!

**Comment by linki** [comment]
wait, I have to update the docs as well...

**Comment by linki** [comment]
alright, updated the docs.

**Pull request merged**